cupy/cuda/compiler.py

```diff
@@ -145,6 +145,17 @@ def _get_arch():
         return min(arch, _nvrtc_max_compute_capability)
 
 
+def _get_arch_for_options(arch=None, jitify=False):
```

Comment:
Better name for this? 😃 
```suggestion
def _get_arch_option_for_nvrtc(arch=None, jitify=False):
```